lucet-spectest/src/error.rs

```diff
@@ -1,6 +1,7 @@
-use failure::{Backtrace, Context, Fail};
-use std::fmt::{self, Display};
+//use std::fmt::{self, Display};
```

Comment:
There's still some cleanup to do here as I track down the handful of spec tests that aren't passing.